components/launch-party/FundingPools.tsx

```diff
@@ -1,4 +1,4 @@
-import BondingPool from "./BondingPool";
```

Comment:
I don't think the original issue was for the launch party Bonding Pools to be renamed to staking. The whole idea was for the other feature to not be confused with this one. So I think all of these on the launch-party should be rollbacked. @pavlovcik what's your take?